libs/shared/ui/src/lib/components/inputs/input-checkbox/input-checkbox.tsx

```diff
@@ -60,15 +60,15 @@ export function InputCheckbox(props: InputCheckboxProps) {
         onChange={(e) => inputChange(e.currentTarget.checked, e)}
         className={`input-checkbox relative font-icons w-0 h-0 appearance-none before:absolute before:flex before:justify-center before:items-center before:text-white before:top-0 before:left-0 before:-translate-y-1/2 before:rounded-sm before:bg-white dark:before:bg-element-light-darker-100 ${bigClasses} ${
           disabled
-            ? 'before:border-element-light-lighter-500'
+            ? 'before:border-element-light-lighter-500 dark:before:border-element-light-lighter-700'
```

Comment:
Its gonna conflict with color name change no?

Reply:
ATM, I didn't touch token with `element` colors, I'll add it soon 👍